script.js

```diff
@@ -94,4 +94,135 @@ function closing() {
+function validationForm() {
+  const firstName = document.forms.desktopForm.firstName.value;
+  const secondName = document.forms.desktopForm.lastName.value;
+  const emailDesk = document.forms.desktopForm.desktopEmail.value;
+  const desktopMess = document.forms.desktopForm.desktopMessage.value;
+
+  let firstText = true;
+  let secondText = true;
+  let emailText = true;
+  let messageText = true;
+
+  if (firstName === '') {
+    errorMessage('firstText', '*Please Enter Your First Name');
+  } else {
+    const nameRegex = /^[a-zA-Z\s]+$/;
+    if (nameRegex.test(firstName) === false) {
+      errorMessage('firstText', '*Please Enter A Valid  First Name');
+    } else {
+      errorMessage('firstText', '');
+      firstText = false;
+    }
+  }
+
+  if (secondName === '') {
+    errorMessage('secondText', '*Please Enter Your Last Name');
+  } else {
+    const lastRegex = /^[a-zA-Z\s]+$/;
+    if (lastRegex.test(secondName) === false) {
+      errorMessage('secondText', '*Please Enter A Valid Last Name');
+    } else {
+      errorMessage('secondText', '');
+      secondText = false;
+    }
+  }
+
+  if (emailDesk === '') {
+    errorMessage('emailText', '*Please enter your email address');
+  } else {
+    const emailRegex = /^[a-zA-Z]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/;
+    if (emailRegex.test(emailDesk) === false) {
+      errorMessage('emailText', '*Please enter a valid email address');
+    } else {
+      errorMessage('emailText', '');
+      emailText = false;
+    }
+  }
+
+  if (desktopMess === '') {
+    errorMessage('messageText', '*Please Enter Your Last Name');
+
+  } else {
+    const messRegex = /^[a-zA-Z\s]+$/;
+    if (messRegex.test(desktopMess) === false) {
+      errorMessage('messageText', '*Please Enter A Valid Last Name');
+    } else {
+      errorMessage('messageText', '');
+      messageText = false;
+    }
+  }
+
+  if ((firstText || secondText || emailText || messageText === true)) {
+    return false;
+  }
+}
+
+validationForm();
+
+function errorMobileMessage (elemId, requiredMessage) {
+  document.getElementById(elemId).innerHTML = requiredMessage;
+}
+
+function validationMobileForm() {
+ const mobileName = document.forms.mobileForm.mobileName.value;
+ const mobileEmail = document.forms.mobileForm.mobileEmail.value;
+ const mobileMessage = document.forms.mobileForm.mobileMessage.value;
+
+ let nameError  = true;
+ let emailError = true;
+ let messageError = true;
+
+  if (mobileName == '') {
+    errorMobileMessage('nameError', '*Please Enter Your First Name');
+  }
+  else {
+    const nameRegex = /^[a-zA-Z\s]+$/;
+    if(nameRegex.test(mobileName) === false){
+      errorMobileMessage('nameError', '*Please Enter A Valid  First Name');
+    } else {
+      errorMobileMessage('nameError', '');
+     nameError = false;
+    }
+  }
+
+  if (mobileEmail == '') {
+    errorMobileMessage('emailError', "*Please enter your email address")
+  } else {
+    const emailRegex = /^[a-zA-Z]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/;
```

Comment:
Hello team ✋

- [ ] [Optional] Please kindly check your validation approach. I, unfortunately, can't seem to submit my message on both the desktop and mobile forms. I can see a validation error on the desktop form but I can't see the same error on the mobile form. 
- Mobile form 
![mobile validation](https://user-images.githubusercontent.com/63917725/180238129-244bf6e9-ed9d-4fd3-833a-e25709d4ac06.PNG)

- Desktop form
![email validation](https://user-images.githubusercontent.com/63917725/180238234-abb72f58-d546-479d-a9db-ee28e244e29e.PNG)

Reply:
Thank you for this review the issue was on the regex validation and now it's fixed the form can be submitted.